system/doc/reference_manual/macros.xml

```diff
@@ -134,7 +134,8 @@ bar(X) ->
 
   <section>
     <title>Predefined Macros</title>
-    <p>The following macros are predefined:</p>
+    <p><marker id="predefined-macros"/>
```

Comment:
The title should auto-generate a marker with that name, so it should not be needed.

Reply:
Ah, good to know!